File: 217. Contains_Duplicate/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        
        unordered_set<int> count_set;
        
        for(int i = 0 ; i < nums.size() ; i++){
            if(count_set.count(nums[i]))
                return true;
            count_set.insert(nums[i]);
        }
        
        return false;
        
    }
};

File: 217. Contains_Duplicate/solution.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        
        Set<Integer> count_set = new HashSet<Integer>();
        
        for(int i = 0 ; i < nums.length ; i++){
            if(count_set.contains(nums[i]))
                return true;
            count_set.add(nums[i]);
        }
        
        return false;
        
        
    }
}